linux/cpu_hot_add_basic/cpu_hot_add_basic.yml

```diff
@@ -15,14 +15,23 @@
   tasks:
     - name: "Test case block"
       block:
+        - name: "Skip test case '{{ ansible_play_name }}'"
+          include_tasks: ../../common/skip_test_case.yml
+          vars:
+            skip_msg: >-
+              Skip test case '{{ ansible_play_name }}' because vCPU hotadd is not supported
+              on ARM ESXi host.
+            skip_reason: "Not Supported"
+          when: esxi_cpu_vendor == 'arm'
+
         # Except for VM guest_id like 'fedora64Guest', 'freebsd.*Guest' not supporting vCPU hot-add,
         # the other guest OS compatible with ansible-vsphere-gos-validation support vCPU hot-add
-        - name: "Skip test case '{{ ansible_play_name }}' due to VM not support vCPU hot-add"
+        - name: "Skip test case '{{ ansible_play_name }}'"
           include_tasks: ../../common/skip_test_case.yml
           vars:
             skip_msg: >-
-              Skip test case '{{ ansible_play_name }}' because vCPU hot-add is not supported
-              by VM configured with guest ID {{ vm_guest_id }}
+              Skip test case '{{ ansible_play_name }}' because vCPU hotadd is not supported
```

Comment:
`hotadd` is not a dictionary word. I think it's better to use `hot add`.

Reply:
Changed in these 2 test cases, while used in other places, we can change it when see it. Thanks.

Reply:
I didn't expect all `hotadd` to be updated in these cases either. You have done a lot. Thanks!